docs/content/preview/deploy/manual-deployment/system-config.md

```diff
@@ -112,7 +112,7 @@ On CentOS, /etc/security/limits.d/20-nproc.conf must also be configured to match
 {{< note title="Note" >}}
 
 If you're using [systemd](https://systemd.io/) to start the processes, and the ulimits are not propagated, you
- must add them also in the `Service` section in the configuration file.
+ must add them also in the `Service` section in the configuration file. See example config for systemd [yb-master.service](/code-samples/yb-master.service) & [yb-tserver.service](/code-samples/yb-tserver.service).
```

Comment:
If you're using [systemd](https://systemd.io/) to start the processes, and the ulimits are not propagated, add the ulimts in the `Service` section of the configuration file. For more details , see systemd example configuration for [yb-master.service](/code-samples/yb-master.service) , and [yb-tserver.service](/code-samples/yb-tserver.service).

Reply:
Added in https://github.com/yugabyte/yugabyte-db/pull/18977/commits/a1b4c2189eeb00a4731421e5723c2bca88fa1e72